src/interactions/map_city/enterMapApartment.interaction.js

```diff
@@ -0,0 +1,7 @@
+export const enterMapApartmentInteraction = (player, k) => {
+    player.onCollide('apartment_door_1', () => {
```

Comment:
@BrianCheung1 did you mean to use one of the houses in the bottom right of the map?
![campus_house_tags](https://github.com/user-attachments/assets/b3af4133-8878-4aa6-89d1-d925b737b880)

If so, double check the tag because there is no apartment_door_1 tag available until now.
Also, it would be cool if you can add a asset so that the door looks like it is open.

Reply:
i used the apartment as the housing for the player I must have overlooked that there were actual houses on the bottom right. 

would you like me to change it to be the bottom housings instead?

![image](https://github.com/user-attachments/assets/4d7b9272-8aa2-44f5-a50b-5b7da798e1c1)

Reply:
@BrianCheung1 that would be great. I thought of the houses in the bottom right section to serve as student apartments from the academy 😅

Reply:
no problem i can do those changes now,
one other issues is that there currently is no spawn point for the houses I believe so I used spawn_arcade before

do I just leave it as that until one is made for the houses?

Reply:
if i wanted to add an asset to look like the door was open, would I need to edit map_city.json using tiled?

Reply:
> no problem i can do those changes now, one other issues is that there currently is no spawn point for the houses I believe so I used spawn_arcade before
> 
> do I just leave it as that until one is made for the houses?

Sure, you can use the arcade or the academy spawn point for now and we can fix it later on.

Reply:
> if i wanted to add an asset to look like the door was open, would I need to edit map_city.json using tiled?

You could just draw a black, light opaque rectangle at the position, or you can look for a sprite which you can add to the public folder directly and load it from there.

Reply:
i was able to add it in, 
could you link where the assets for the restroom sink/toilet are
I'm almost done with the interior design of the house but unable to find where u guys got those assets

Reply:
Sorry I forgot adding the reference for you. Have a look at [the asset credits](https://github.com/zero-to-mastery/ZTM-Quest/blob/main/asset_credits.md) where we added everything what we have used until now.

Reply:
Added open door for campus_house_1
added interior with spawn point, walls, boundaries, interactive objects
added reference image for interactive objects to additional map info readme

Reply:
🔥🔥🔥